async-http-client/src/main/scala/org/http4s/client/asynchttpclient/AsyncHttpClient.scala

```diff
@@ -26,7 +26,7 @@ object AsyncHttpClient {
   val defaultConfig = new DefaultAsyncHttpClientConfig.Builder()
     .setMaxConnectionsPerHost(200)
     .setMaxConnections(400)
-    .setRequestTimeout(30000)
+    .setRequestTimeout(60000)
```

Comment:
This raises an interesting philosophical point: should there be an http4s default for all client backends, that's as similar as possible, or should we default to the backends' defaults?

Reply:
I think we should set our defaults, since http4s is the principal abstraction here. - users would probably be surprised if changing the backend, while staying within http4s would change defaults.

Reply:
I think that's a reasonable position.

Maybe we should have a defaults object in client to centralize all that, and then adapt as well as we can in each of the backends.

Reply:
I like a second PR to do the latter. As I think there will likely be more than this that we will want to change in that PR. But I think it would be a good idea.